doc/content/source/postprocessors/RMSDistance.md

```diff
@@ -0,0 +1,15 @@
+# RMSDistance
+
+!syntax description /Postprocessors/RMSDistance
+The RMSDistance postprocessor computes the RMS distance of the variable parameter $v$ from a fixed point $\vec{r}_0$:
```

Comment:
The lines after the description command only render if you add an empty line here!

http://mooseframework.inl.gov/magpie/docs/PRs/313/site/source/postprocessors/RMSDistance.html

Reply:
Since this does not respect periodic boundary conditions you should add an `!alert warning` box to point that out (or use `minPeriodicVector` to make it respect PBCs).

Reply:
> Since this does not respect periodic boundary conditions
RMSDistance will accumulate whatever the variable value is at any point in the domain regardless of whether it is periodic. Magpie would remap anything that leaves the domain back onto it and _does_ not keep track of where the contribution actually is in physical space. To get the correct RMS distribution of say a single damage cascade, you can hence not use periodic boundary conditions. I don't see how `minPeriodicVector` helps.

Reply:
Ok I see what `minPeriodicVector` does. The question is if RMSDistance is meaningful if periodic boundaries are present.